package.json

```
@@ -3,18 +3,17 @@
   "version": "0.7.0",
   "repository": "vercel/node-file-trace",
   "license": "MIT",
-  "main": "./out/node-file-trace.js",
-  "types": "./out/node-file-trace.d.ts",
+  "main": "./out/index.js",
+  "types": "./out/index.d.ts",
   "bin": {
     "nft": "./out/cli.js"
   },
   "scripts": {
     "build": "tsc",
     "prepublishOnly": "tsc && rm out/tsconfig.tsbuildinfo",
     "test": "jest --verbose",
-    "test-verbose": "jest --verbose --coverage --globals \"{\\\"coverage\\\":true}\"",
-    "codecov": "codecov",
-    "test-coverage": "jest --verbose --coverage --globals \"{\\\"coverage\\\":true}\" && codecov"
+    "test-verbose": "tsc --sourceMap && jest --verbose --coverage --globals \"{\\\"coverage\\\":true}\"",
```

I think this should work too:

```suggestion
    "test-verbose": "tsc --sourceMap && jest --verbose --coverage --globals '{\"coverage\":true}'",
```